package.json

```diff
@@ -105,29 +105,32 @@
     "postcss-preset-env": "^7.0.1",
     "postcss-resolution-independence": "^1.0.1",
     "postcss-safe-parser": "^6.0.0",
-    "prompts": "2.4.1",
-    "react": "17.0.1",
+    "prompts": "^2.4.2",
+    "react": "^17.0.2",
     "react-dev-utils": "^12.0.0",
-    "react-dom": "17.0.1",
-    "react-refresh": "0.10.0",
-    "react-test-renderer": "17.0.1",
-    "resolution-independence": "1.0.0",
-    "resolve": "1.20.0",
-    "semver": "7.3.5",
+    "react-dom": "^17.0.2",
+    "react-refresh": "^0.11.0",
+    "react-test-renderer": "^17.0.2",
+    "resolution-independence": "^1.0.0",
+    "resolve": "^1.20.0",
+    "semver": "^7.3.5",
     "source-map-loader": "^3.0.0",
-    "strip-ansi": "6.0.0",
-    "style-loader": "3.0.0",
-    "tar": "6.0.2",
-    "terser-webpack-plugin": "5.2.5",
-    "v8-compile-cache": "2.1.1",
-    "validate-npm-package-name": "3.0.0",
-    "webpack": "5.64.4",
-    "webpack-dev-server": "4.6.0"
+    "strip-ansi": "^6.0.1",
+    "style-loader": "^3.0.0",
+    "tailwindcss": "^3.0.2",
+    "tar": "^6.1.11",
+    "terser-webpack-plugin": "^5.2.5",
+    "v8-compile-cache": "^2.3.0",
+    "validate-npm-package-name": "^3.0.0",
+    "webpack": "^5.64.4",
+    "webpack-dev-server": "^4.6.0"
+
   },
   "optionalDependencies": {
-    "fsevents": "^2.1.3"
+    "fsevents": "^2.3.2"
   },
   "peerDependencies": {
+    "react": ">= 16",
```

Comment:
🤔 Why is this added? and why it's 16?

Reply:
It's the same with CRA. from CRA`s #9872 PR.
I think this change is just for providing the latest react, a minimum 16.
(in the description of CRA: `react-scripts` has a missing peer dependency on `react`)